Makefile

```diff
@@ -13,6 +13,7 @@ JAVA_GEN_VERSION := v0.7.1
 TESTPARALLELISM := 10
 WORKING_DIR := $(shell pwd)
 SCHEMA_TOOLS_VERSION := $(shell cat .schema-tools.version)
+PULUMI_PROVIDER_BUILD_PARALLELISM ?= -p 2
```

Comment:
This is a behavior change from -p 1 but @thomas11 recommended this. We can make it -p 1. Still lets me run with -p 6 locally.